homeassistant/components/lektrico/button.py

```diff
@@ -0,0 +1,104 @@
+"""Support for Lektrico buttons."""
+
+from collections.abc import Callable, Coroutine
+from dataclasses import dataclass
+from typing import Any
+
+from lektricowifi import Device
+
+from homeassistant.components.button import (
+    ButtonDeviceClass,
+    ButtonEntity,
+    ButtonEntityDescription,
+)
+from homeassistant.const import ATTR_SERIAL_NUMBER, CONF_TYPE, EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import LektricoConfigEntry, LektricoDeviceDataUpdateCoordinator
+from .entity import LektricoEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class LektricoButtonEntityDescription(ButtonEntityDescription):
+    """Describes Lektrico button entity."""
+
+    press_fn: Callable[[Device], Coroutine[Any, Any, dict[Any, Any]]]
+
+
+BUTTONS_FOR_CHARGERS: tuple[LektricoButtonEntityDescription, ...] = (
+    LektricoButtonEntityDescription(
+        key="charge_start",
+        translation_key="charge_start",
+        entity_category=EntityCategory.CONFIG,
+        press_fn=lambda device: device.send_charge_start(),
+    ),
+    LektricoButtonEntityDescription(
+        key="charge_stop",
+        translation_key="charge_stop",
+        entity_category=EntityCategory.CONFIG,
+        press_fn=lambda device: device.send_charge_stop(),
+    ),
+    LektricoButtonEntityDescription(
+        key="reboot",
+        translation_key="reboot",
+        device_class=ButtonDeviceClass.RESTART,
+        entity_category=EntityCategory.CONFIG,
+        press_fn=lambda device: device.send_reset(),
+    ),
+)
+
+BUTTONS_FOR_LB_DEVICES: tuple[LektricoButtonEntityDescription, ...] = (
+    LektricoButtonEntityDescription(
+        key="reboot",
+        translation_key="reboot",
```

Comment:
```suggestion
        device_class=ButtonDeviceClass.RESTART,
        entity_category=EntityCategory.CONFIG,
        press_fn=lambda device: device.send_reset(),
    ),
)

BUTTONS_FOR_LB_DEVICES: tuple[LektricoButtonEntityDescription, ...] = (
    LektricoButtonEntityDescription(
        key="reboot",
```

Reply:
I removed translation_key="reboot", thank you.